apps/estuary/test/integration/event_table_test.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Estuary.EventTableTest do
```

Comment:
Personally, I'd move your other create table test into this file. You could also put all 3 tests in the same file, which would speed up the test run time (though I like that you broke them out). I think I'd also prefer that you do the same  show tables check as it'd be easy to just return `[[true]]` from `EventTable.create_table()` to make the test pass. Doing the show tables really proves to me that the table is (still) there.

Reply:
I agree with you to put all the create table test to EventTableTest file but the reason I've left 2 tests in the ApplicationStartTest file is that it will show those test belongs to startup (in future it may be easy to understand) and the 1 test in EventTableTest is to make sure if someone has broke the create table function which belongs to EventTable, then it will be useful in that scenario. Let me know your thoughts.

Reply:
Regarding `Show Table`, as per my understanding it is already tested in the `ApplicationStartTest` and it may be duplicated and for `EventTable.create_table()` it should always return true because we are using `IF NOT EXISTS`, we may avoid the other call to the database by firing the other `SHOW TABLE` query

Reply:
Generally I think more along the lines of 'what are we doing' than 'when are we doing things' in an app. If I care about the table creator I'd go to the table creator test as opposed to the 'startup test', so it makes more sense to me to have all tests regarding table creation in the table creator test as opposed to a startup test. Just my 2 cents.

EventTable.create_table() is returning the result of a prestige call. That call may change its return value which would break this test (even though it may still be doing its job correctly). Also, I don't care _what_ prestige returns, I care that the table (still) _exists_. So I think checking the outcome is more robust / explicit than checking the return value.

Reply:
Added one more assert for `show table`